engine/ddf/src/ddf/ddf.h

```diff
@@ -51,6 +52,8 @@ namespace dmDDF
         Descriptor* m_MessageDescriptor;
         uint32_t    m_Offset;
         const char* m_DefaultValue;
+        uint8_t     m_OneOfIndex : 7;
+        uint8_t     m_OneOfSet   : 1;
```

Comment:
We _could_ merge these or just use the 'm_OneOfSet' to distinguish if a field is part of a oneof definition